.husky/commit-msg

```diff
@@ -1,11 +1,4 @@
-#!/usr/bin/env sh
-
-# https://typicode.github.io/husky/guide.html#disable-husky-in-ci-docker-prod
-[ -n "$CI" ] && exit 0
```

Comment:
I moved this functionality to the `env` section of the relevant GitHub workflows